dxr/testing.py

```diff
@@ -207,30 +208,45 @@ class SingleFileTestCase(TestCase):
     # print its path so you can examine it:
     should_delete_instance = True
 
+    # Override this in a subclass to change the plugins used.
+    enabled_plugins = ('pygmentize', 'clang')
+
+    # Override this in a subclass to change the build command.
+    # $CXX gets injected by the clang DXR plugin:
+    build_command = '$CXX -o main main.cpp'
```

Comment:
Seems like SingleFileTestCase should be an abstract class, and there should be concrete C and Python subclasses that fill these out. Does have to happen in this PR; could follow up later.